src/native-common/Image.tsx

```diff
@@ -179,10 +195,27 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
         if (this.props.headers) {
             source.headers = this.props.headers;
         }
+        if (this._forceCache) {
```

Comment:
This looks to me like it's actually state. The only reason you make it here on changing is by calling forceUpdate, which is somewhat of an anti-pattern anyways

Reply:
I wanted to avoid introducing state for perf reasons. Browsing through the rxp sources it doesn't look that uncommon to track state in fields and call forceUpdate.

Reply:
I was under the wrong impression that stateless components have better perf characteristics. Converted to state as suggested, thanks!